<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelTabSubtileWidget</class>
 <widget class="QWidget">
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="solAndSptHorizontalLayout">
     <item>
      <widget class="QGroupBox" name="solSettingsGroupBox">
       <property name="title">
        <string>Collision settings</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="sol0">
          <property name="text">
           <string>Solid</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="sol1">
          <property name="text">
           <string>Block light</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="sol2">
          <property name="text">
           <string>Block missile</string>
          </property>
         </widget>
        </item>
        <!--item row="0" column="3">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item-->
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="sptSettingsGroupBox">
       <property name="title">
        <string>Special settings</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel">
          <property name="text">
           <string>Light Radius:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="lightComboBox">
          <property name="toolTip">
           <string>Inherent light radius of this subtile</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel">
          <property name="text">
           <string>Trap:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="trapNoneRadioButton">
          <property name="text">
           <string>None</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">trapButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QRadioButton" name="trapLeftRadioButton">
          <property name="text">
           <string>Left</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">trapButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QRadioButton" name="trapRightRadioButton">
          <property name="text">
           <string>Right</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">trapButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel">
          <property name="text">
           <string>Spec:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QComboBox" name="specCelComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
          <property name="toolTip">
           <string>Special cel-frame assigned to this subtile</string>
          </property>
         </widget>
        </item>
        <!--item row="0" column="5">
         <widget class="LineEditWidget" name="specCelLineEdit">
          <property name="charWidth">
           <number>2</number>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
          <property name="toolTip">
           <string>Special cel-frame assigned to this subtile</string>
          </property>
         </widget>
        </item-->
        <item row="0" column="7">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="tmiSettingsGroupBox">
     <property name="title">
      <string>Render settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="tmi0">
        <property name="text">
         <string>Transp. Wall</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel">
        <property name="text">
         <string>Left floor:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="tmi1">
        <property name="text">
         <string>Second pass</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="tmi2">
        <property name="text">
         <string>Foliage</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="tmi3">
        <property name="text">
         <string>Transparent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel">
        <property name="text">
         <string>Right floor:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="tmi4">
        <property name="text">
         <string>Second pass</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="tmi5">
        <property name="text">
         <string>Foliage</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="tmi6">
        <property name="text">
         <string>Transparent</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="smpSettingsGroupBox">
     <property name="title">
      <string>Map settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Map feature:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="smpTypeComboBox">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Extern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stairs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>West Door</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>East Door</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel">
        <property name="text">
         <string>Walls:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="smp4">
        <property name="toolTip">
         <string>North-West</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="smp5">
        <property name="toolTip">
         <string>North-East</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="smp6">
        <property name="toolTip">
         <string>South-West</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QCheckBox" name="smp7">
        <property name="toolTip">
         <string>South-East</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Frames:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="framesPrevButton">
       <property name="minimumWidth">
        <number>40</number>
       </property>
       <property name="maximumWidth">
        <number>40</number>
       </property>
       <property name="toolTip">
         <string>Previous frame</string>
       </property>
       <property name="text">
        <string notr="true">&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="framesComboBox">
       <property name="minimumWidth">
        <number>80</number>
       </property>
       <property name="maximumWidth">
        <number>120</number>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtCurrent</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="framesNextButton">
       <property name="minimumWidth">
        <number>40</number>
       </property>
       <property name="maximumWidth">
        <number>40</number>
       </property>
       <property name="toolTip">
        <string>Next frame</string>
       </property>
       <property name="text">
        <string notr="true">&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="trapButtonGroup"/>
 </buttongroups>
</ui>
